src/mvc/Model.js

```diff
@@ -0,0 +1,56 @@
+export default class Model {
+  constructor() {
+    this.API = "https://jsonplaceholder.typicode.com";
+    this.posts = [];
+    this.selectedPost = null;
+  }
+
+  async fetchPosts() {
+    try {
+      const allPosts = await fetch(`${this.API}/posts`);
+      const allComments = await fetch(`${this.API}/comments`);
+      let posts = await allPosts.json();
+      posts = posts.slice(0, 10);
+      let comments = await allComments.json();
+
+      for (let i = 0; i < posts.length; i++) {
```

Comment:
real layihələrdə for`lardan istifade etmirler, üstünlüyü HOF verilir, kodda for`ları görəndə belə nəticəyə gəlirəm ki Array metodlarını zəif bilirsən

Reply:
const postsWithComments = posts.map(post => {
  const postComments = comments.filter(comment => comment.postId === post.id);
  return {
    ...post,
    comments: postComments,
  };
});
this.posts.push(postsWithComments);
----------------
bele yaziram amma ishlemir.